content/content-strategy/cms/cms-requirements/section8.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: text/textblock
+---
+Most sites and CMSs have 3 levels of content management, each with a different purpose. 
+
+These are:
+- the development or test site – where changes and new features are built and tested.
+- the staging site – a mirror version of your live site which lets you preview pages before they go live.
+- the production site – your public facing live site.
+
+A staging and live site may be enough for most agencies in terms of pure content management. How these areas work together can have a big impact on the long term maintenance of your CMS. 
+
+If you want to develop your website in the future, you’ll need to be clear on how your CMS will handle changes. This includes things like navigation, functionality and new content requirements.
+
+Consider the following:
+- how many development and staging environments may be needed
+- how data and files flow between ‘staging’ and ‘live’ environments
+- how developers will pull down a production version of your site, that has been stripped of personal or secure information
+- if the CMS puts any restrictions on your preferred deployment methods
+
+### [1]Preview capabilities
```

Comment:
Incorrect display level. Needs to be `### [2]` to display a heading 3 correctly.

Reply:
Steve. Fixed